client/lib/abtest/active-tests.js

```diff
@@ -101,5 +101,14 @@ module.exports = {
 			description: 50
 		},
 		defaultVariation: 'description'
-	}
+	},
+	headstart: {
+		datestamp: '20160203',
+		variations: {
+			original: 20,
```

Comment:
This seems like a lot of users to send through the test; can we do this instead?

```
headstart: {
	datestamp: '20160203',
	variations: {
		main: 20,
		headstart: 20,
		notTested: 60
	},
	defaultVariation: 'main'
},
```